Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/HolographicCamera/HolographicCameraOriginAnchor.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer;
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.WorldAnchors;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using UnityEngine.XR.WSA;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.HolographicCamera
+{
+    [RequireComponent(typeof(WorldAnchorManager))]
+    [RequireComponent(typeof(TCPConnectionManager))]
+    public class HolographicCameraOriginAnchor : MonoBehaviour
+    {
+        private const string AnchorName = "WorldRoot";
+        private WorldAnchorManager worldAnchorManager;
+        private TCPConnectionManager tcpConnectionManager;
+
+        public bool IsAnchorLocated
+        {
+            get
+            {
+                WorldAnchor anchor = GetComponent<WorldAnchor>();
+                return anchor != null && anchor.isLocated;
+            }
+        }
+
+        private void Start()
+        {
+            worldAnchorManager = GetComponent<WorldAnchorManager>();
+            worldAnchorManager.AttachAnchor(gameObject, AnchorName);
+
+            tcpConnectionManager = GetComponent<TCPConnectionManager>();
+            tcpConnectionManager.OnReceive += TcpConnectionManager_OnReceive;
+        }
```

Comment:
nit: based on other subscriptions, it seems this should have an unsubscription on destroy.

Reply:
Especially when subscribing to a singleton.

Another nit: initialization should happen on awake, as start is much later in the lifecycle.